src/types_unstable.ts

```diff
@@ -0,0 +1,54 @@
+export type * from './aggregate.js';
```

Comment:
1. Should we put a comment in this file to the effect of "Types exported from this file are considered unstable, and may be changed/removed at any time" ? 

2. How did you determine which files to export from? (I.e. when we add new files to Vega-Lite, what would drive us to update this file)? If we wanted to, we could probably prune this list down to a smaller number (maybe just the ones that are in the root or end with index.js). If the idea is that every file under `src` except itself should be listed, we could handle that with a comment first, and look into scripting/linting it later.